### PR TITLE
Fixed adding subhandlers in logging tests, better cleanup in LogCategoriesTestCase

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LoggingAbstractTestCase.java [hunk 1]
```diff
@@ -117,4 +119,20 @@ protected void verifyIfErrorAppears(String identifier, String value) {
     protected final String getTmpDirPath(String subdir) {
         return new File(System.getProperty("java.io.tmpdir"), subdir).getAbsolutePath();
     }
+
+    protected void addHandlers(Address address, String attributeName) throws Exception {
+        String handler_one = "log-cat-test-handler_" + RandomStringUtils.randomAlphanumeric(5);
+        String handler_two = "log-cat-test-handler_" + RandomStringUtils.randomAlphanumeric(5);
+        client.apply(   Logging.handler().console().add(handler_one).build(),
+                        Logging.handler().console().add(handler_two).build());
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LoggingAbstractTestCase.java]: Any reason for the spaces after `(` ?

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LoggingAbstractTestCase.java [hunk 2]
```diff
@@ -117,4 +119,20 @@ protected void verifyIfErrorAppears(String identifier, String value) {
...
+        administration.reloadIfRequired();
+        try {
+            editTextAreaAndVerify(address, attributeName, new String[]{handler_one, handler_two});
+        } finally {
+            operations.undefineAttribute(address, attributeName);
+            client.apply(   Logging.handler().console().remove(handler_one),
+                            Logging.handler().console().remove(handler_two));
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/logging/LoggingAbstractTestCase.java]: Any reason for the spaces after `(` ?

**Comment by rhatlapa** [comment]
There are spaces after `(` which I believe shouldn't be there. Otherwise it looks ok.

**Comment by honza-kasik** [comment]
Removed unnecessary space, sorry, too much Python...

**Comment by rhatlapa** [comment]
LGTM, please squash.

**Comment by honza-kasik** [comment]
Squashed and force pushed, thanks.

**Comment by rhatlapa** [comment]
Thanks, merging.

**Pull request merged**